### PR TITLE
EWS,ILS: comparison report change

### PR DESCRIPTION
**Pull request by kkrampa**
Now reports are showing only records that are not migrated.

### CODE REVIEW

File: custom/ilsgateway/comparison_reports.py [hunk 1]
```diff
@@ -112,19 +107,17 @@ def rows(self):
             web_users.extend(chunk)
 
         for web_user in web_users:
-            is_migrated = True
             try:
                 user = User.objects.get(username__in=[web_user.username, web_user.email.lower()])
                 webuser = WebUser.get_by_username(user.username)
-                if webuser:
-                    is_migrated = self.domain in webuser.get_domains()
+                if webuser and not self.domain in webuser.get_domains():
```
Comment (dimagimon) [inline custom/ilsgateway/comparison_reports.py]: E713 test for membership should be 'not in'

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by czue** [comment]
looks good - but can we just remove the `is_migrated` flag entirely from the array?

**Comment by czue** [comment]
(won't block on it)

**Pull request merged**